playbook/app/pb_kits/playbook/pb_body/_body.scss

```diff
@@ -12,13 +12,22 @@
       }
     }
   }
-
   @each $dark_color_name, $dark_color_value in $pb_dark_body_colors{
     &[class*=_#{$dark_color_name}][class*=dark]{
       @include pb_body($dark_color_value);
     }
   }
+  b, strong {
+    letter-spacing: -.03em;
+    font-weight: 700;
```

Comment:
2 things:

1. Always use our tokens if they are available. If they are not question why they are not and get them created.
2. Why not just include the styling for title 4? [Import this](https://github.com/powerhome/playbook/blob/master/playbook/app/pb_kits/playbook/tokens/_titles.scss) and add `@include pb_title_4;`